src/app/users/[id]/components/UserInfoDetail.tsx

```diff
@@ -0,0 +1,128 @@
+import { Input } from '@/_components/common/Input'
+import { Radio } from '@/_components/common/Radio'
+import { Button } from '@/_components/common/Button'
+import { IcoSearch } from '@/_components/icons'
+import { DatePicker } from '@/_components/common/DatePicker'
+import { Row } from '../components/Row'
+import { Cell, CellLabel } from '../components/Cell'
+import { SearchAddress } from '@/_components/common/SearchAddress'
+
+import { useState } from 'react'
+
+export const UserInfoDetail = () => {
+  const [searchAddressPopup, setSearchAddressPopup] = useState(false)
+  const [address, setAddress] = useState('')
+
+  const handleComplete = () => {
+    setSearchAddressPopup(!searchAddressPopup)
+  }
+
+  return (
+    <section className="border w-full h-full rounded-xl shadow-md">
+      <h2 className="text-xl font-bold mb-4 py-5 px-10 my-3">회원정보</h2>
+      <table className="w-full border-separate border-spacing-y-4 px-8">
+        <tbody className="">
+          <Row>
+            <CellLabel label="아이디" />
+            <Cell>
+              <Input disabled></Input>
+            </Cell>
+          </Row>
+
+          <Row>
+            <CellLabel label="비밀번호" />
+            <Cell>
+              <Input
+                type="password"
+                placeholder="비밀번호를 입력해주세요"
+              ></Input>
```

Comment:
이건 그냥 제 스타일이긴 한데 전달하는 children이 없는 경우 아래처럼 컴포넌트를 바로 닫아도 괜찮을 것 같습니다!

```suggestion
              <Input
                type="password"
                placeholder="비밀번호를 입력해주세요"
              />
```

Reply:
딩크군여